src/sui-autocompleted/results-list.jsx

```diff
@@ -1,23 +1,60 @@
-import React from 'react'
+/* eslint-disable */
+import React, {Fragment} from 'react'
 import PropTypes from 'prop-types'
 import ListItem from './list-item'
 
-export default function ResultsList (props) {
+export default function ResultsList ({
+    active,
+    getSectionSuggestions,
+    handleSelect,
+    renderSection,
+    renderSuggestion,
+    suggests,
+    withSections
```

Comment:
What about `false` like default value? It can improve readability.

Reply:
done